### PR TITLE
Register new package TakingBroadcastSeriously.jl v0.1.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [MikeInnes/TakingBroadcastSeriously.jl](https://github.com/MikeInnes/TakingBroadcastSeriously.jl)
Release: [v0.1.0](https://github.com/MikeInnes/TakingBroadcastSeriously.jl/releases/tag/v0.1.0)
cc: @MikeInnes

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

@MikeInnes This PR will remain open for 24 hours for feedback (which is optional). If you get feedback, please let us know if you are making changes, and we'll merge once you're done.

### CODE REVIEW

**Comment by tkelman** [comment]
would travis pass if enabled?

**Comment by tkelman** [comment]
should fill in the repo description to a better summary than "No description, website, or topics provided."

**Comment by tkelman** [comment]
Please fill in a repo description.

File: TakingBroadcastSeriously/versions/0.1.0/requires [hunk 1]
```diff
@@ -0,0 +1 @@
+julia 0.5
```
Comment (tkelman) [inline TakingBroadcastSeriously/versions/0.1.0/requires]: this uses `struct` syntax and doesn't support 0.5

please pay attention to these things, they matter. blatantly ignoring me and all aspects of versioning your releases doesn't accomplish anything except broken code.

Comment (omus) [reply TakingBroadcastSeriously/versions/0.1.0/requires]: Additionally the line:
```julia
using Base.Broadcast: broadcast_c, containertype
```
Won't work in Julia 0.5 as these functions don't exist in Base.

**Pull request merged**